src/components/beta/OnboardingChecklist.tsx

```
@@ -0,0 +1,441 @@
+            <div className="w-6 h-6 rounded-full bg-alpine-blue-500 flex items-center justify-center">
+              <SparklesIcon className="w-4 h-4 text-white" />
+            </div>
+            <span className="font-semibold text-sm text-granite-900 dark:text-granite-100">
+              Base Camp
+            </span>
+          </div>
+          <button
+            onClick={handleDismiss}
+            className="text-granite-400 hover:text-granite-600 dark:hover:text-granite-200 transition-colors"
+            aria-label="Dismiss checklist"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)